### DISCUSSION TITLE
Passing workspace/folder via Apache configuration

### DISCUSSION THREAD

**Discussion by bilogic** [coder/code-server#5758] [Q&A]
Is it possible to specify which folder `code-server` should open using Apache's configuration?

I host a number of projects on different domains on my Apache and would like to edit that particular project when accessing via its domain. If this can be done, then it is possible to only run 1 instance of code-server, saving resources.

**Comment by jsjoeio** [comment]
I haven't used Apache before but you could specify the folder via the query param like this:


```
https://localhost:8080/?folder=/Users/jp/dev/coder
```
